intlogger.go

```diff
@@ -401,22 +405,31 @@ func (l *intLogger) logPlain(t time.Time, name string, level Level, msg string,
 			if strings.Contains(val, "\n") {
 				l.writer.WriteString("\n  ")
 				l.writer.WriteString(key)
-				l.writer.WriteString("=\n")
 				if l.fieldColor != ColorOff {
-					writeIndent(l.writer, val, faintColor.Sprint("  | "))
+					l.writer.WriteString(faintColor.Sprint("=\n"))
+					writeIndent(l.writer, val, faintColor.Sprint(multiLinePrefix))
```

Comment:
I meant pulling the whole `faintColor.Sprint()` output into a global var, so that we're not recalculating it on every call. Ditto with the colorization of "=\n" on line 409.

Reply:
Should be addressed with https://github.com/hashicorp/go-hclog/pull/118/commits/fd264152857468feff6a784c047e6585bebdd664